test/cert-ceremonies/root-ceremony-rsa.yaml

```diff
@@ -20,3 +20,14 @@ certificate-profile:
     key-usages:
         - Cert Sign
         - CRL Sign
+skip-lints:
+   - e_ext_authority_key_identifier_missing
+   - e_ext_authority_key_identifier_no_key_identifier
+   - e_sub_ca_aia_does_not_contain_ocsp_url
+   - e_sub_ca_aia_missing
+   - e_sub_ca_certificate_policies_missing
+   - e_sub_ca_crl_distribution_points_missing
+   - n_ca_digital_signature_not_set
+   - n_mp_allowed_eku
```

Comment:
This lint from the mozilla root program Section 5.3 only applies to intermediates.